src/services/link/index.js

```diff
@@ -27,7 +37,14 @@ module.exports = function setup() {
 						const [url, query] = data.target.split('?');
 						const queryObject = queryString.parse(query || '');
 						queryObject.link = data._id;
-						res.redirect(`${url}?${queryString.stringify(queryObject)}`);
+						if (url.includes('/registration/') && !(verifyDate(data.createdAt))) {
```

Comment:
Here we should take care that the link is internal btw. the url path should start with `/registration`

Reply:
there's something odd about how links are handled there; I'll check when I have time